packages/kwc-pagination/kwc-pagination.js

```diff
@@ -16,6 +16,7 @@ class KWCPagination extends PolymerElement {
                 justify-content: center;
                 flex-direction: row;
                 align-items: center;
+                margin-bottom: 40px;
```

Comment:
If the component is broken / displaying wrong because of this missing margin, then :+1:. But if it's to space it out from the surrounding elements, I think it would be better to set the style by the parent that instantiates it (and is aware of the context) for better reusability.

Reply:
E.g.,

```ts
static get styles() {
    return css`
         kwc-pagination { margin-bottom: 40px }
    `;
}

render() {
    return html`
        <kwc-pagination></kwc-pagination>
        <div class="other-stuff">
        </div>
    `;
}
```